src/main/resources/assets/aiotbotania/lang/en_us.json

```diff
@@ -27,6 +27,8 @@
   "item.aiotbotania.elementium_hoe": "Elementium Hoe",
   "item.aiotbotania.elementium_aiot": "Elementium AIOT",
 
+  "item.aiotbotania.terra_aiot": "Terrasteel AIOT",
+
```

Comment:
lexica botania entry is missing

Reply:
I did not get this work. I dont know what I'm doing wrong but it just does not work.